functions.py

```diff
@@ -0,0 +1,162 @@
+import csv
+import datetime
+import exceptions_tests
+import os #this is needed for using directory paths and manipulating them
+import sys
+from pip._vendor.distlib.compat import raw_input
+
+class Employee(object):
+
+    def __init__(self, employee_id, name, phone, age):
+        self.employee_id = employee_id
+        self.name = name
+        self.phone = phone
+        self.age = age
+
+class Employees_list(object):
+    def __init__(self, list_of_employees):
+        self.list_of_employees = list_of_employees
+
+    def add_employee(self, new_employee):
+        """
+        This function gets the credentials of a new employee from user and adds it to the employees file.
+        """
+        with open(self.list_of_employees, mode='a') as csvFile:
```

Comment:
If the employee is already existing?

Reply:
I tried to add a check to the function but it runs properly without doing the check, I'd be happy if you could take a look and see what's wrong in my command:

```    def add_employee(self, new_employee):
        new_employee_data = [new_employee.employee_id, new_employee.name, new_employee.phone, new_employee.age]
        with open(self.list_of_employees) as file1:
            existing_lines = csv.reader(file1)
            for row in existing_lines:
                if new_employee_data not in row:
                    new_employee_data.append(row)
                else:
                    print("Sorry, the employee is already exist.")
```

Reply:
done :)